src/templates/partials/footer-items.njk

```diff
@@ -0,0 +1,25 @@
+<section class="footer__wrapper">
+        <ul class="footer__list">
+          {% for nav in nav  %}
+              <li class="footer__item">
+                  {% if nav.name == "Subscribe" %}
+                  <a class="footer__link footer__link--medium" href="{{ nav.link }}">{{ nav.name }}</a>
+                  {% else  %}
+                  <a class="footer__link" href="{{ nav.link }}">{{ nav.name }}</a>
+                  {% endif %} 
+              </li>
+          {% endfor %}
+        </ul>
+
+        <ul class="footer__list">
+          {% for info in info  %}
```

Comment:
How about to change some name, because it a little bit confused? Now I understand my mistake. Might you renamed it after my comment about `faqs`?